pkgs/development/libraries/dotherside/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub, cmake, qtbase, qtquickcontrols2, wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "dotherside";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "filcuc";
+    repo = "DOtherside";
+    rev = "v${version}";
+    sha256 = "sha256-o6RMjJz9vtfCsm+F9UYIiYPEaQn+6EU5jOTLhNHCwo4=";
+  };
+
+  buildInputs = [ qtbase qtquickcontrols2 ];
+  nativeBuildInputs = [ cmake wrapQtAppsHook ]; 
+
+  meta = with lib; {
+    description = "A C language library for creating bindings for the Qt QML language";
+    homepage = "https://filcuc.github.io/dotherside";
+    license = licenses.lgpl3;
```

License seems to be [`LGPLv3+`](https://github.com/filcuc/dotherside/blob/master/lib/src/DOtherSideTypesCpp.cpp#L7-L9)

```suggestion
    license = licenses.lgpl3Plus;
```

---

Yeah, I wasn't sure just by the `LICENSE` file I looked at.